### PR TITLE
Add API calls to manage group/project badges

### PR DESCRIPTION
**Pull request by fhaefemeier**
Enhancement #402 implemented. It add API functions to get, edit, add and remove badges of groups and projects. Unit tests implemented.

### CODE REVIEW

File: spec/gitlab/client/projects_spec.rb [hunk 1]
```diff
@@ -66,8 +142,6 @@
     end
 
     it 'returns information about a created project' do
-      expect(@project.name).to eq('Gitlab')
-      expect(@project.owner.name).to eq('John Smith')
```
Comment (NARKOZ) [inline spec/gitlab/client/projects_spec.rb]: Why are these deleted?

Comment (fhaefemeier) [reply spec/gitlab/client/projects_spec.rb]: Don't knbow, it was a mistake. I add it back.

**Comment by fhaefemeier** [comment]
I checked the lint job messages. I have trouble to fix the message:
`spec/gitlab/client/projects_spec.rb:20:86: C: Style/FormatStringToken: Prefer annotated tokens (like %<foo>s) over template tokens (like %{foo}).
      expect(@badges.first.link_url).to eq('http://example.com/ci_status.svg?project=%{project_path}&ref=%{default_branch}')
`. Do you know how it can be changed in a way to satisfy rubocop? I am not so familar with it. Is it possible to quote the string in some way, so it will no longer recognized as tokens?

**Comment by NARKOZ** [comment]
ping @fhaefemeier

**Comment by fhaefemeier** [comment]
> ping @fhaefemeier

Sorry, I hadn't time to fix the issues yet. I will try to do it in the next days.
